train_hierarchialXC.py

```diff
@@ -23,8 +28,14 @@
 # training configuration arguments
 parser.add_argument('--seed', type=int, default=None,
                     help='Manually set the seed for the experiments for reproducibility')
-parser.add_argument('--plot', action='store_true',
-                    help='Option to plot the loss variation over iterations')
+parser.add_argument('--interval', type=int, default=-1,
+                    help='Interval between two status updates on training')
+parser.add_argument('--input_ae_dim', type=int, default=500,
+                    help='Dimensionality of encoding in the input encoder')
+parser.add_argument('--input_ae_factor', type=float, default=0.25,
```

Comment:
Irrelevant.

Reply:
Let me elaborate, I was under the impression that we'll set this parameter permanently.